### PR TITLE
Update address

### PR DESCRIPTION
**Pull request by DeepDiver1975**
- [x] requires #61

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
rebased

**Comment by irgendwie** [comment]
Type (Home, Work, Other) of address isn't saved. :-1:
The rest looks good!

**Comment by DeepDiver1975** [comment]
grrrr - on changing the type the data structure is also changed from an array to an object - wtf

File: js/components/detailsItem/detailsItem_controller.js [hunk 1]
```diff
@@ -3,7 +3,12 @@ app.controller('detailsItemCtrl', ['$templateRequest', 'vCardPropertiesService',
 
     ctrl.meta = vCardPropertiesService.getMeta(ctrl.name);
     ctrl.t = {
-        country : t('contacts', 'Country'),
+        poBox : t('contactsrework', 'Post Office Box'),
```
Comment (nickvergessen) [inline js/components/detailsItem/detailsItem_controller.js]: Wrong app name

File: js/public/script.js [hunk 2]
```diff
@@ -357,7 +358,12 @@ app.controller('detailsItemCtrl', ['$templateRequest', 'vCardPropertiesService',
 
     ctrl.meta = vCardPropertiesService.getMeta(ctrl.name);
     ctrl.t = {
-        country : t('contacts', 'Country'),
+        poBox : t('contactsrework', 'Post Office Box'),
```
Comment (nickvergessen) [inline js/public/script.js]: wrong app name

**Comment by DeepDiver1975** [comment]
please retest @irgendwie @nickvergessen THX

**Comment by irgendwie** [comment]
@DeepDiver1975 merge conflict

**Comment by DeepDiver1975** [comment]
> @DeepDiver1975 merge conflict

will rebase

**Comment by DeepDiver1975** [comment]
rebased

**Comment by irgendwie** [comment]
LGTM :+1:

**Comment by DeepDiver1975** [comment]
thx

**Pull request merged**